docs/development.md

```diff
@@ -101,7 +101,7 @@ $ TAG=<tag> REGISTRY=$USER/ingress-controller make image
 ## Deploying
 
 There are several ways to deploy the ingress controller onto a cluster.
-Please check the [deployment guide](./deploy/)
+Please check the [deployment guide](../deploy/)
```

Comment:
Why are you changing this link? This points to the deploy directory inside docs

Reply:
I’m not sure if it's the markdown parser problem. But the current link is indeed 404.

Reply:
Maybe the following files should be put inside the directory.

- development.md
- how-it-works.md
- troubleshooting.md
- kubectl-plugin.md

Reply:
@agile6v no without proper redirections to not break existing links

Reply:
Using `./deploy/index.md` works. Please change it and test locally running `make live-docs`

Reply:
It works.